Tribler/Core/Session.py

```diff
@@ -123,6 +123,19 @@ def set_and_create_dir(dirname, setter, default_dir):
                 permidmod.save_keypair(self.keypair, pairfilename)
                 permidmod.save_pub_key(self.keypair, pubfilename)
 
+            multichain_pairfilename = scfg.get_multichain_permid_keypair_filename()
+
+            if os.access(multichain_pairfilename, os.F_OK):
```

Comment:
Good call. I based this on the existing code for the regular permid, but os.path.exists() should be an improvement there as well.